src/System.Security.Cryptography.OpenSsl/tests/project.json

```diff
@@ -19,10 +19,27 @@
     "Microsoft.DotNet.BuildTools.TestSuite": "1.0.0-prerelease-00704-03"
   },
   "frameworks": {
-    "netstandard1.6": {}
+    "netstandard1.6": {},
+    "netstandard1.7": {}
   },
   "supports": {
     "coreFx.Test.netcoreapp1.0": {},
-    "coreFx.Test.net463": {}
+    "coreFx.Test.net463": {},
+    "coreFx.Test.netcoreapp1.1-ns17": {
+        "netstandard1.7": [
+            "win7-x86",
+            "win7-x64",
```

Comment:
Probably not. Copy-paste from a (working) json. I'll remove it, but not sure if this is the new explicit style we are now required for xunit stuff, or if something's not just ready yet. Anyways it should make the project.lock.json smaller